nrniv/Parallel/alltoall.py

```diff
@@ -39,7 +39,7 @@ def serialize():
 
 if rank == 0:
   from os import system
-  print (system('cmp temp alltoall.cmp'))
+  print((system('cmp temp alltoall.cmp')))
```

Comment:
Why the extra ()? Doesn't seem to make a difference.

Reply:
apparently `-p` needs to be specified: 
```
When the -p is passed, 2to3 treats print as a function instead of a statement. This is useful when from __future__ import print_function is being used. If this option is not given, the print fixer will surround print calls in an extra set of parentheses because it cannot differentiate between the print statement with parentheses (such as print ("a" + "b" + "c")) and a true function call.
```
https://docs.python.org/3.0/library/2to3.html

Reply:
I will however run `black` on the python files.